.github/workflows/nightly.yml

```diff
@@ -353,5 +353,21 @@ jobs:
       pr_no: '0'
       bench_script_params: '--save Baseline_PVC'
 
+  Benchmarks-sycl:
+    uses: ./.github/workflows/reusable_benchmarks.yml
+    permissions:
+      contents: write
+      pull-requests: write
+    with:
+      pr_no: '0'
+      bench_script_params: >-
+        --adapter level_zero_v2
+        --compute-runtime
+        --build-igc
+        --preset Minimal
+        --exit-on-failure
+        --save Baseline_Arc
+      runner: 'L0_PERF_GPU'
```

Comment:
hmm this label is a little unprecise - I've added new ones (for this PC it's L0_PERF_ARC, for the other machine, you can update it as well, please - L0_PERF_PVC)

Reply:
That's better, right